dnf-behave-tests/features/repoquery/main.feature

```diff
@@ -673,7 +672,6 @@ Given I successfully execute dnf with args "install bottom-a1"
 # --querytags
 @bz1744073
 # not with os is related to 1879168
```

Comment:
you probably should remove also this comment

Reply:
right